src/lib.rs

```diff
@@ -478,6 +480,8 @@ pub mod pallet {
 						// the native account has already some rewards in, we add the new ones
 						AccountsPayable::<T>::insert(
 							native_account,
+							// It should be safe not to use saturating_add here
```

Comment:
I think it is better to still use the saturating_add in those cases no? Is there a cost to it ?

Reply:
A small one in terms of a couple of checks (the value being greater than the maximum). But I guess they are minimal, so I can also add it here